dali/test/dali_operator_test.h

```diff
@@ -27,26 +27,139 @@
 namespace dali {
 namespace testing {
 
-using Arguments = std::map<ArgumentKey, double>;  // TODO(mszolucha) some generalization boost::any?
+using Arguments = std::map<ArgumentKey, int>;  // TODO(mszolucha) some generalization boost::any?
```

Comment:
Will you add the generalization to dali::Value in this PR?

Reply:
Yup, just figuring out a way to use it